src/index.js

```diff
@@ -1,13 +1,14 @@
 import Telegraf from 'telegraf';
+const botName = process.env.BOT_NAME;
+const app = new Telegraf(process.env.BOT_TOKEN, { username: botName });
 import debugMiddleware from './middlewares/debug';
 import photoUploadMiddleware from './middlewares/photoUpload';
+import tos from './middlewares/termsOfService';
 
-const app = new Telegraf(
-    process.env.BOT_TOKEN,
-    { username: process.env.BOT_USERNAME }
-);
+console.log('token', process.env.BOT_TOKEN);
 
 app.use(debugMiddleware);
 app.use(photoUploadMiddleware);
+app.use(tos);
 
-app.startPolling();
+app.startPolling(0);
```

Comment:
This is the amount of time between each poll, in seconds, as described [here](http://telegraf.js.org/telegraf.html#startpolling).